HW0/P6/P6B.py

```diff
@@ -16,16 +16,22 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [10**(x/10.0) for x in range(-60,1,1)]
 
     for t in wait_time:
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
-
+        parallelStart = time.time()
+        pool.map(burnTime(t), range(N))
```

Comment:
I don't think it is correct here. 
You want pool.map(burnTime, [t]*N)
[t]*N is the vector on which you map, you want it to wait 16 times during t (not during range(16))
But it is a good try!

I think that is why you get a speedup up to 16, it should not be better than 4 since you only have 4 processors